plugins/ua_pubsub_ethernet.c

```diff
@@ -192,7 +196,7 @@ UA_PubSubChannelEthernet_open(const UA_PubSubConnectionConfig *connectionConfig)
     struct sockaddr_ll sll = { 0 };
     sll.sll_family = AF_PACKET;
     sll.sll_ifindex = channelDataEthernet->ifindex;
-    sll.sll_protocol = htons(ETHERTYPE_UADP);
+    sll.sll_protocol = htons(ETH_P_802_2);
```

Comment:
Why this change?

As far as I can see in the spec, this was correct before.

Try have a look at https://reference.opcfoundation.org/v104/Core/docs/Part14/7.3.3/